joint_names:
  ["base_cyl",
  "cyl_arm1",
  "arm1_arm2",
  "arm2_arm3",
  "arm3_tool"]
joint_ranges:
  [[-2.0, 2.0],
  [-1.45, 0.79],
  [0.0, 2.36],
  [-1.45, 0.79],
  [-3.14, 3.14]]
joy_ranges:
  [[-1.0, 1.0],
  [-1.0, 1.0],
  [-1.0, 1.0],
  [-1.0, 1.0],
  [-1.0, 1.0]]
